src/main/java/se/sundsvall/emailreader/integration/db/AttachmentEntity.java

```diff
@@ -0,0 +1,52 @@
+package se.sundsvall.emailreader.integration.db;
+
+
+import java.time.LocalDateTime;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.PrePersist;
+import jakarta.persistence.Table;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.Generated;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "attachment")
+@Data
+@Builder(setterPrefix = "with")
+@NoArgsConstructor
+@AllArgsConstructor(access = AccessLevel.PROTECTED)
+public class AttachmentEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "content")
+    private String content;
+
+    @Column(name = "content_type")
+    private String contentType;
+
+    @Column(name = "created_at")
+    private LocalDateTime createdAt;
+
+    @Generated // To avoid jacoco coverage check for this field
```

Comment:
Same outcome, but use `@ExcludeFromJacocoGeneratedCoverageReport` instead, mainly for clarity?